File: capture_hark.js
const playwright = require('playwright');
const fs = require('fs');

(async () => {
  const browser = await playwright.chromium.launch();
  
  // Desktop screenshots
  const desktopContext = await browser.newContext({
    viewport: { width: 1280, height: 800 }
  });
  const desktopPage = await desktopContext.newPage();
  await desktopPage.goto('https://hark.ryan.ceo/', { waitUntil: 'networkidle' });
  
  // Wait a bit for any animations
  await desktopPage.waitForTimeout(3000);
  
  // Viewport screenshot
  await desktopPage.screenshot({ path: '/opt/cursor/artifacts/screenshots/hark-desktop.png' });
  
  // Full page screenshot
  await desktopPage.screenshot({ path: '/opt/cursor/artifacts/screenshots/hark-desktop-full.png', fullPage: true });
  
  // Extract HTML content for analysis
  const htmlContent = await desktopPage.content();
  fs.writeFileSync('/opt/cursor/artifacts/screenshots/hark-page-content.html', htmlContent);
  
  await desktopContext.close();
  
  // Mobile screenshots
  const mobileContext = await browser.newContext({
    viewport: { width: 390, height: 844 }
  });
  const mobilePage = await mobileContext.newPage();
  await mobilePage.goto('https://hark.ryan.ceo/', { waitUntil: 'networkidle' });
  
  // Wait a bit for any animations
  await mobilePage.waitForTimeout(3000);
  
  await mobilePage.screenshot({ path: '/opt/cursor/artifacts/screenshots/hark-mobile.png' });
  
  await mobileContext.close();
  await browser.close();
  
  console.log('Screenshots captured successfully!');
})();

File: capture_hark.py
from playwright.sync_api import sync_playwright
import time

def capture_screenshots():
    with sync_playwright() as p:
        browser = p.chromium.launch()
        
        # Desktop screenshots
        print("Capturing desktop screenshots...")
        desktop_context = browser.new_context(viewport={'width': 1280, 'height': 800})
        desktop_page = desktop_context.new_page()
        desktop_page.goto('https://hark.ryan.ceo/', wait_until='networkidle')
        
        # Wait for animations and interactions
        time.sleep(5)
        
        # Viewport screenshot
        desktop_page.screenshot(path='/opt/cursor/artifacts/screenshots/hark-desktop.png')
        print("✓ Desktop viewport screenshot saved")
        
        # Full page screenshot
        desktop_page.screenshot(path='/opt/cursor/artifacts/screenshots/hark-desktop-full.png', full_page=True)
        print("✓ Desktop full-page screenshot saved")
        
        # Extract HTML content
        html_content = desktop_page.content()
        with open('/opt/cursor/artifacts/screenshots/hark-page-content.html', 'w') as f:
            f.write(html_content)
        print("✓ HTML content saved")
        
        desktop_context.close()
        
        # Mobile screenshots
        print("Capturing mobile screenshots...")
        mobile_context = browser.new_context(viewport={'width': 390, 'height': 844})
        mobile_page = mobile_context.new_page()
        mobile_page.goto('https://hark.ryan.ceo/', wait_until='networkidle')
        
        # Wait for animations
        time.sleep(5)
        
        mobile_page.screenshot(path='/opt/cursor/artifacts/screenshots/hark-mobile.png')
        print("✓ Mobile screenshot saved")
        
        mobile_context.close()
        browser.close()
        
        print("\nAll screenshots captured successfully!")

if __name__ == "__main__":
    capture_screenshots()
